src/components/related/ProductCard.jsx

```diff
@@ -61,7 +62,9 @@ class ProductCard extends React.Component {
       axios(pricePicsRequestConfig)
         .then((response) => {
           const data = response.data.results[0];
-          this.setState({price: data.original_price, salePrice: data.sale_price});
+          this.setState({price: data.original_price,
```

Comment:
I would add an extra newline before the first item in this object

Reply:
That does look cleaner! I'll push the change in a moment